### PR TITLE
fix(c/libuv): Fix return type FsType & struct rename

### PR DESCRIPTION
**Pull request by hackerchai**
Signed-off-by: hackerchai <i@hackerchai.com>

### CODE REVIEW

File: c/libuv/libuv.go [hunk 1]
```diff
@@ -297,7 +297,7 @@ func (handle *Handle) Fileno(fd *OsFd) c.Int {
 }
 
 //go:linkname UvPipe C.uv_pipe
-func UvPipe(fds [2]UvFile, readFlags c.Int, writeFlags c.Int) c.Int {
+func UvPipe(fds [2]File, readFlags c.Int, writeFlags c.Int) c.Int {
```
Comment (aofei) [inline c/libuv/libuv.go]:
```suggestion
func Pipe(fds [2]File, readFlags c.Int, writeFlags c.Int) c.Int {
```

Comment (hackerchai) [reply c/libuv/libuv.go]: There is a repeated Struct named `pipe`, so we rename the func into `UvPipe`
- `libuv.go`
```go
type Pipe struct {
	Unused [0]byte
}
```

Comment (xushiwei) [reply c/libuv/libuv.go]: maybe: type `uv_pipe_t` => `PipeT`, func  `uv_pipe` => `Pipe`.

Comment (hackerchai) [reply c/libuv/libuv.go]: > maybe: type `uv_pipe_t` => `PipeT`, func `uv_pipe` => `Pipe`.

solved due to srtuct `Pipe`removed.

**Pull request merged**